bundles/org.openhab.binding.tapocontrol/src/main/java/org/openhab/binding/tapocontrol/internal/constants/TapoThingConstants.java

```diff
@@ -63,6 +65,7 @@ public class TapoThingConstants {
     public static final String DEVICE_DESCRIPTION_SMART_CONTACT = "Smart-Contact-Sensor";
     public static final String DEVICE_DESCRIPTION_MOTION_SENSOR = "Motion-Sensor";
     public static final String DEVICE_DESCRIPTION_TEMP_SENSOR = "Temperature-Sensor";
+    public static final String DEVICE_DESCRIPTION_SMART_SWITCH = "Smart Switch";
```

Comment:
looking at the other device descriptions, i would expect a dash instead of a whitespace to be consistent.

Reply:
Thx for reviewing. I will change it